dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -37,17 +37,48 @@ warning.could.not.thin.application=CWWKC0251W: Could not convert the application
 warning.could.not.thin.application.explanation=The application could not be converted to a thin application. The application will continue to run from the original JAR. 
 warning.could.not.thin.application.useraction=Attempt to resolve the issue in the exception message and either restart the server or update the application. 
 
-error.need.springboot.version.15=CWWKC0252E: The application failed to start because the springBoot-2.0 feature is configured in the server.xml file. The application requires the springBoot-1.5 feature to be configured.
-error.need.springboot.version.15.explanation=The application requires the springBoot-1.5 feature. Check the version of the Spring Boot feature that is currently configured in the server.xml file.  
-error.need.springboot.version.15.useraction=Configure the springBoot-1.5 feature in the server.xml file. Remove any other Spring Boot features.
-
-error.need.springboot.version.20=CWWKC0253E: The application failed to start because the springBoot-1.5 feature is configured in the server.xml file. The application requires the springBoot-2.0 feature to be configured.
-error.need.springboot.version.20.explanation=The application requires the springBoot-2.0 feature. Check the version of the Spring Boot feature that is currently configured in the server.xml file.  
-error.need.springboot.version.20.useraction=Configure the springBoot-2.0 feature in the server.xml file. Remove any other Spring Boot features.
-
-error.missing.servlet=CWWKC0254E: The application failed to start because it requires a servlet feature to be configured in the server.xml file.
-error.missing.servlet.explanation=The application is a web application, and it requires a servlet feature to be configured in the server.xml file.
-error.missing.servlet.useraction=Configure a servlet feature in the server.xml file.
+#UPDATED for SpringBoot 3.0:
+# error.need.springboot.version.15=CWWKC0252E: The application failed to start because the springBoot-2.0 feature is configured in the server.xml file. The application requires the springBoot-1.5 feature to be configured.
+#error.need.springboot.version.15.explanation=The application requires the springBoot-1.5 feature. Check the version of the Spring Boot feature that is currently configured in the server.xml file.  
+#error.need.springboot.version.15.useraction=Configure the springBoot-1.5 feature in the server.xml file. Remove any other Spring Boot features.
+
+#0: The required spring feature.
+#1: The current spring feature.
+#2: The version of spring content found in the application.
+error.spring.required.15=CWWKC0252E: Error: Feature {0} must be provisioned: Feature {1} is provisioned and the application has Spring {2} content.
```

Comment:
what is "provisioned" in this context? We generally talk about features as being "enabled" or "configured". Unless "provisioned" has specific meaning, I'd recommend using "configured". Otherwise, need to explain what "provisioned" means in this context

Reply:
Changing "provisioned" to "configured".  (I've been using "provisioned" to mean listed as a feature in the server configuration.)